subsys/bluetooth/controller/ll_sw/ull_df.c

```diff
@@ -81,6 +81,8 @@ static uint8_t mem_link_iq_report_quota_pdu;
 #if defined(CONFIG_BT_CTLR_DF_ADV_CTE_TX)
 static struct lll_df_adv_cfg lll_df_adv_cfg_pool[CONFIG_BT_CTLR_ADV_AUX_SET];
 static void *df_adv_cfg_free;
+static uint8_t remove_cte_info_from_per_adv_chain(struct ll_adv_set *adv,
+						  struct lll_df_adv_cfg *df_cfg, uint8_t *ter_idx);
```

Comment:
Move `uint8_t *ter_idx` to the next line`

Reply:
Although it looks wired, I suggest to leave it as is. This is a result of use of clang-format. It does not hurt but makes our lives easier without enable/disable automatic formatting in future.

Reply:
From https://www.kernel.org/doc/html/latest/process/coding-style.html: 

> Descendants are always substantially shorter than the parent and are placed substantially to the right. A very commonly used style is to align descendants to a function open parenthesis.

Basically what this say, is that when breaking long lines, the lines should be shorts than the previous (when applicable of course).

Reply:
Zephyr's checkpatch was aligned to that and reports warnings after exceeding 100 characters limit. .clang-format was also aligned to that and approved by community. So I assume that it is our limit now.

Reply:
Moved to new line.